### DISCUSSION TITLE
The infrastructure for creating a 125 terabyte S3 storage

### DISCUSSION THREAD

**Discussion by aoberest** [seaweedfs/seaweedfs#2337] [Q&A] [answered]
Hello. 
I received a request to calculate the infrastructure for creating a 125 terabyte S3 storage.
Please give me some advice. 
How to calculate the number of nodes ?
How much RAM should I choose for one node?
Which seaweedfs image should I use large or normal? 
I plan to use 3 filer with shared storage on postgresql - how do you like this option?

**Comment by chrislusf** [comment, marked as answer]
You can start small. Just need to ensure the filer store can sustain for a while. Filers, Masters, and Volume Servers can be added later if capacity is running low.

Filer store depends on your use case. Whether the files are constantly changing, how much is the access QPS, etc.

Use the Seaweed large image to have some head room.
